catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/DumpCommand.java

```diff
@@ -265,7 +279,24 @@ protected Object executeWithSubject() throws Exception {
     }
 
     if (StringUtils.isNotBlank(zipFileName)) {
-      createZip(catalog, queryRequest, dirPath + zipFileName, resultCount);
+      File outputFile = new File(dirPath + zipFileName);
+      createZip(catalog, queryRequest, outputFile, resultCount);
+
+      String alias = System.getProperty(SystemBaseUrl.EXTERNAL_HOST);
+      String password = System.getProperty("javax.net.ssl.keyStorePassword");
+
+      try (InputStream inputStream = new FileInputStream(outputFile)) {
+        byte[] signature = signer.createDigitalSignature(inputStream, alias, password);
+
+        if (signature == null) {
+          console.println("An error occurred while signing export");
+        } else {
+          String epoch = Long.toString(Instant.now().getEpochSecond());
```

Comment:
Any special reason to use epoch seconds vs something like iso8601 version without invalid path chars? eg `20190319T205342Z`

Reply:
I didn't choose epoch seconds for any particular reason other than that I'm familiar with it. By appending the epoch seconds onto the file name I'm trying to make it so it's likely unique. 

I'll change it to ISO-8601 though.